docs/DeveloperGuide.md

```diff
@@ -377,7 +386,7 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 #### 7.4.2 Design consideration:
 
-##### 7.4.2.1 Aspect: How undo executes
+#### 7.4.2.1 Aspect: How undo executes
```

Comment:
![image](https://user-images.githubusercontent.com/10459011/98539566-87f8d500-22c7-11eb-942f-abf51319e7cf.png)

Please help to put information source on new line if not it will not render as well